From 7254f6c52b5da38c0a79ab953d34e556fe16942f Mon Sep 17 00:00:00 2001
From: Sebastian Hesselbarth <sebastian.hesselbarth@gmail.com>
Date: Fri, 19 Sep 2014 21:27:55 +0200
Subject: ARM: mvebu: armada-xp: Add GE0 pinctrl settings for GMII

There is a GMII setting for GE0, add it to the common pinctrl node.

Signed-off-by: Sebastian Hesselbarth <sebastian.hesselbarth@gmail.com>
Tested-By: Benoit Masson <yahoo@perenite.com>
Signed-off-by: Jason Cooper <jason@lakedaemon.net>

--- a/arch/arm/boot/dts/armada-xp.dtsi
+++ b/arch/arm/boot/dts/armada-xp.dtsi
@@ -75,6 +75,17 @@
 			pinctrl: pin-ctrl@18000 {
 				reg = <0x18000 0x38>;
 
+				pmx_ge0_gmii: pmx-ge0-gmii {
+					marvell,pins =
+					     "mpp0",  "mpp1",  "mpp2",  "mpp3",
+					     "mpp4",  "mpp5",  "mpp6",  "mpp7",
+					     "mpp8",  "mpp9",  "mpp10", "mpp11",
+					     "mpp12", "mpp13", "mpp14", "mpp15",
+					     "mpp16", "mpp17", "mpp18", "mpp19",
+					     "mpp20", "mpp21", "mpp22", "mpp23";
+					marvell,function = "ge0";
+				};
+
 				pmx_ge0_rgmii: pmx-ge0-rgmii {
 					marvell,pins =
 					     "mpp0", "mpp1", "mpp2", "mpp3",
